src/Sulu/Bundle/DocumentManagerBundle/Initializer/RootPathPurger.php

```diff
@@ -50,10 +50,11 @@ public function purge()
         $sessions = $this->connections->getConnections();
         $rootPath = '/' . $this->pathSegments->getPathSegment($this->rootRole);
 
-        foreach ($sessions as $session) {
-            if ($session->nodeExists($rootPath)) {
-                $session->getNode($rootPath)->remove();
-                $session->save();
+        foreach ($sessions as $name => $session) {
```

Comment:
Yes. The admin connection exists for each session, i.e. you *could* (I
don't know why you *would*) have a DBAL PHPCR session and a Jackrabbit
session. Each would have a corresponding "admin connection", as far as I
understand it.

On Mon, Mar 14, 2016 at 10:03:09AM -0700, Daniel Rotter wrote:
>    In
>    [1]src/Sulu/Bundle/DocumentManagerBundle/Initializer/RootPathPurger.php:
> 
>  > @@ -50,10 +50,11 @@ public function purge()
>  >          $sessions = $this->connections->getConnections();
>  >          $rootPath = '/' . $this->pathSegments->getPathSegment($this->rootRole);
>  >
>  > -        foreach ($sessions as $session) {
>  > -            if ($session->nodeExists($rootPath)) {
>  > -                $session->getNode($rootPath)->remove();
>  > -                $session->save();
>  > +        foreach ($sessions as $name => $session) {
> 
>    Is this loop still needed?
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/2125#discussion_r56035666
>    2. https://github.com/sulu-io/sulu/pull/2125/files#r56035666
>    3. https://github.com/sulu-io/sulu/pull/2125/files#r56035666